FindMissingNumber.py

```diff
@@ -0,0 +1,21 @@
+# Time Complexity : O(logn)
+# Space Complexity : O(1)
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : No
+
+# Perform binary search. If value at index -minus value at mid is not consistent then that means some number has gone missing.
+
```

Comment:
instead just check the value at mid if it is less than target move to right side else left side